lib/plugins/aws/invokeLocal/index.js

```diff
@@ -122,11 +122,20 @@ class AwsInvokeLocal {
     if (runtime.startsWith('nodejs')) {
       const handlerPath = handler.split('.')[0];
       const handlerName = handler.split('.')[1];
-      return this.invokeLocalNodeJs(
-        handlerPath,
-        handlerName,
-        this.options.data,
-        this.options.context);
+      if (runtime.endsWith('8.10'))  {
```

Comment:
I think this does _NOT_ work. Awaiting a promise or awaiting the callback must be independent of the configured Node runtime. You can freely decide if you use async/await or the callback with each Node runtime.

**Invoke local should just check if the execution of the handler function returns a promise or not**. In case it returns a promise, then await it, otherwise listen for the callback. BTW: AWS Node 6 also supports returning a promise.